intake/gui/source/gui.py

```diff
@@ -112,9 +112,11 @@ def visible(self, visible):
     def callback(self, sources):
         """When a source is selected, enable widgets that depend on that condition
         and do done_callback"""
+        if hasattr(self, 'plot'):
+            # guard since this cannot happen until plot is ready
+            self.plot.visible = False
```

Comment:
I don't think this needs to be here. Setting `self.plot_widget.value = False` should do it.

Reply:
That is exactly what already exists two lines below, but doesn't have the right effect.

Reply:
Huh I thought that would be enough. But if not I guess it's fine.